packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -368,6 +371,14 @@ export const preparePortfolioKit = (
           const { portfolioId } = this.state;
           publishStatus(makeFlowStepsPath(portfolioId, id), steps);
         },
+        publishFlowOneStep(
```

Comment:
I first read this as something about the first step. It's also confusing that `publishFlowSteps` doesn't publish all the steps. There's an ambiguity between `steps` as the array at `.steps` and the set of steps matching `steps.stepN`. We can clarify with "Record" as the thing being published.


```suggestion
        /**
          * Refresh the status for a single step record
          */        
        publishFlowStepRecord(
```

Please also update doc on `publishFlowSteps` to clarify that it updates the single `.steps` node. If you rename that to `publishFlowStepsRecord` it will be clear that it's updating one record instead of plural. (vs `publishFlowStepRecords`)

Reply:
Rolled back, since I implemented @dckc's `pendingTxs` and treeish `flowSteps` for dependent transactions instead of creating a new vstorage path per step.